<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MissionControlTab</class>
 <widget class="QWidget" name="MissionControlTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Map View</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="vLayout"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Mission Control Panel</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Select Robot</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <layout class="QVBoxLayout" name="robotsL"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Actions</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QPushButton" name="pauseBtn">
               <property name="text">
                <string>Pause Action</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="captureImage">
               <property name="text">
                <string>Capture Image</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="generateSearchSpace">
               <property name="text">
                <string>Generate Search Space</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pathPlanBtn">
               <property name="text">
                <string>Path Plan</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pathFollowBtn">
               <property name="text">
                <string>Path Follow</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="intentionRecognitionBtn">
               <property name="text">
                <string>Start IRecognition</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="resetBeliefBtn">
               <property name="text">
                <string>Reset Belief</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="connect2Robot">
               <property name="palette">
                <palette>
                 <active>
                  <colorrole role="WindowText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>0</red>
                     <green>0</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Button">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>0</red>
                     <green>255</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Light">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>127</red>
                     <green>255</green>
                     <blue>127</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Midlight">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>63</red>
                     <green>255</green>
                     <blue>63</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Dark">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>0</red>
                     <green>127</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Mid">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>0</red>
                     <green>170</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Text">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>0</red>
                     <green>0</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="BrightText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>255</red>
                     <green>255</green>
                     <blue>255</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="ButtonText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>0</red>
                     <green>0</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Base">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>255</red>
                     <green>255</green>
                     <blue>255</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Window">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>0</red>
                     <green>255</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Shadow">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>0</red>
                     <green>0</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="AlternateBase">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>127</red>
                     <green>255</green>
                     <blue>127</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="ToolTipBase">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>255</red>
                     <green>255</green>
                     <blue>220</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="ToolTipText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>0</red>
                     <green>0</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </active>
                 <inactive>
                  <colorrole role="WindowText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>0</red>
                     <green>0</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Button">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>0</red>
                     <green>255</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Light">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>127</red>
                     <green>255</green>
                     <blue>127</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Midlight">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>63</red>
                     <green>255</green>
                     <blue>63</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Dark">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>0</red>
                     <green>127</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Mid">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>0</red>
                     <green>170</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Text">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>0</red>
                     <green>0</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="BrightText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>255</red>
                     <green>255</green>
                     <blue>255</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="ButtonText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>0</red>
                     <green>0</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Base">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>255</red>
                     <green>255</green>
                     <blue>255</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Window">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>0</red>
                     <green>255</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Shadow">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>0</red>
                     <green>0</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="AlternateBase">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>127</red>
                     <green>255</green>
                     <blue>127</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="ToolTipBase">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>255</red>
                     <green>255</green>
                     <blue>220</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="ToolTipText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>0</red>
                     <green>0</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </inactive>
                 <disabled>
                  <colorrole role="WindowText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>0</red>
                     <green>127</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Button">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>0</red>
                     <green>255</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Light">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>127</red>
                     <green>255</green>
                     <blue>127</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Midlight">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>63</red>
                     <green>255</green>
                     <blue>63</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Dark">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>0</red>
                     <green>127</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Mid">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>0</red>
                     <green>170</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Text">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>0</red>
                     <green>127</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="BrightText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>255</red>
                     <green>255</green>
                     <blue>255</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="ButtonText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>0</red>
                     <green>127</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Base">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>0</red>
                     <green>255</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Window">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>0</red>
                     <green>255</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="Shadow">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>0</red>
                     <green>0</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="AlternateBase">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>0</red>
                     <green>255</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="ToolTipBase">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>255</red>
                     <green>255</green>
                     <blue>220</blue>
                    </color>
                   </brush>
                  </colorrole>
                  <colorrole role="ToolTipText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>0</red>
                     <green>0</green>
                     <blue>0</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </disabled>
                </palette>
               </property>
               <property name="text">
                <string>Connect to Robot</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
